.travis.yml

```diff
@@ -8,18 +8,13 @@ branches:
   only:
     - master
 node_js:
+  - "10"
   - "8"
-  - "6"
 services:
   - docker
 env:
   global:
     - COUCHBASE_USER="Administrator"
     - COUCHBASE_PASS="password"
-  matrix:
-    - COUCHBASE="couchbase4"
```

Comment:
can we still keep `matrix` to avoid repeating the test code, I see the cb5 and cb4 test code are almost the same excepts the authentication part.

Reply:
Yes, I will take some time to figure it out